src/EventListener/ExceptionListener.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Symfony\WebpackEncoreBundle\EventListener;
+
+use Symfony\WebpackEncoreBundle\Asset\EntrypointLookup;
+use Symfony\WebpackEncoreBundle\Asset\EntrypointLookupCollection;
+
+class ExceptionListener
+{
+    private $entrypointLookupCollection;
+
+    private $entrypointNames;
```

Comment:
I believe these are `buildNames` - we should change `entrypoint` -> `build` everywhere in this class